webpack.config.views.js

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env node
```

Comment:
What's with this line at the top?

Reply:
This is something the cosmos db team had in their web view config but I don't think it is something we should need.